core/ledger/kvledger/tests/env.go

```diff
@@ -217,7 +218,8 @@ func populateMissingsWithTestDefaults(t *testing.T, initializer *ledgermgmt.Init
 		}
 		cryptoProvider, err := sw.NewDefaultSecurityLevelWithKeystore(sw.NewDummyKeyStore())
 		assert.NoError(t, err)
-		membershipInfoProvider := privdata.NewMembershipInfoProvider(createSelfSignedData(cryptoProvider), identityDeserializerFactory)
+		mspID := viper.GetString("peer.localMspId")
```

Comment:
Do we really need to bring in viper here?  Can't we just set the mspID to a test value?

Reply:
Probably not, this was ported from 1.4 when it was still using viper so I probably overlooked this when cherry-picking. I'll remove it, thanks for catching that!